src/main/java/org/example/Main.java

```diff
@@ -1,9 +1,12 @@
 package org.example;
 
-// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
-// then press Enter. You can now see whitespace characters in your code.
+
+import org.example.models.*;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println();
-        }
-    }
\ No newline at end of file
+
+        PlayerStatus player = new PlayerStatus();
+        player.initPlayer("Alina");
```

Comment:
Pls do not use real person names, use something like "Jane Doe".

Reply:
Comment implemented in [8d226507d](https://github.com/razvanexu/JavaLearn_Player_Status/pull/1/commits/8d226507d7f516d4bf6041bf2ddfa36f508b131c)